stock_warehouse_calendar_orderpoint/models/stock_warehouse.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import fields, models
+
+
+class StockWarehouse(models.Model):
+    _inherit = "stock.warehouse"
+
+    orderpoint_calendar_id = fields.Many2one(
```

Comment:
Could you explain the difference between the `orderpoint_calendar_id` and the [`calendar_id` defined in `stock_warehouse_calendar`](https://github.com/OCA/stock-logistics-warehouse/blob/f13ff658c2e8625ad264701108083610aed6fab8/stock_warehouse_calendar/models/stock_warehouse.py#L12)?

It kind of seems to me that, conceptually, they both refer to the same thing, no?
In which case would you use one but not the other?